trimesh/points.py

```diff
@@ -110,26 +111,33 @@ def radial_sort(points,
       Origin to sort around
     normal : (3,)  float
       Vector to sort around
+    start : (3,) float
+      Vector to specify position of 12 in clock viewing in
+      direction of normal, must not be parallel with normal
 
     Returns
     --------------
     ordered : (n, 3) float
       Same as input points but reordered
     """
 
-    # create two axis perpendicular to each other and the normal,
-    # and project the points onto them
-    axis0 = [normal[0], normal[2], -normal[1]]
-    axis1 = np.cross(normal, axis0)
-    ptVec = points - origin
-    pr0 = np.dot(ptVec, axis0)
-    pr1 = np.dot(ptVec, axis1)
-
+    # create two axis perpendicular to each other and
+    # the normal and project the points onto them
+    if start is None:
+        axis0 = [normal[0], normal[2], -normal[1]]
+        axis1 = np.cross(normal, axis0)
+    else:
+        normal, start = util.unitize([normal, start])
+        if np.abs(1 - np.abs(np.dot(normal, start))) < tol.zero:
+            raise ValueError('start must not parallel with normal')
+        axis0 = np.cross(start, normal)
+        axis1 = np.cross(axis0, normal)
+    vectors = points - origin
     # calculate the angles of the points on the axis
-    angles = np.arctan2(pr0, pr1)
-
+    angles = np.arctan2(np.dot(vectors, axis0),
+                        np.dot(vectors, axis1))
     # return the points sorted by angle
-    return points[[np.argsort(angles)]]
+    return points[angles.argsort()[::-1]]
```

Comment:
Sorting points as `angles` descending is not corresponding to the doc "Vector to specify position of 12 in clock viewing in direction of normal".

Is this a typo?

Reply:
Yeah I just changed that for the unit test so the result would match a theta `np.linspace(0, np.pi*2, 1000)` but don't have a strong opinion (increasing vs decreasing). Happy to match behavior of the docstring (descending theta) or change the docstring to "increasing" if you have a preference.

Reply:
Changing the docstring seems more reasonable: #1582

Reply:
Thanks for the fix!